dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/model/MasterHeartBeat.java

```diff
@@ -36,6 +36,7 @@ public class MasterHeartBeat implements HeartBeat {
     private double reservedMemory;
     private double diskAvailable;
     private int processId;
+    private boolean overload;
```

Comment:
Add Master status rather than overload flag, master may have other status in the future.